src/main/duplicateDetector.ts

```
@@ -265,6 +266,14 @@ export class DuplicateDetector {
 
   private selectHighestQuality(tracks: Track[]): Track {
     return tracks.reduce((best, current) => {
+      const bestLossless = isLossless(best.location);
+      const currentLossless = isLossless(current.location);
+
+      // Lossless always beats lossy, regardless of bitrate or metadata.
+      if (currentLossless && !bestLossless) return current;
```

ESLint

[error] Expected { after 'if' condition.

(curly)